Basic FE project template/ui-src/js/utilities.js

```diff
@@ -0,0 +1,18 @@
+const utilities = {
+  backToTopButton: function () {
+    const button = document.getElementById("btn-scroll");
```

Comment:
bnt-scroll is not a correct way to define and id, ids are written like so: bntScroll, but in this case, you should target by class and create a class that you want to use in javascript, such as "js-scroll".